cmd/guaccollect/cmd/deps_dev.go

```diff
@@ -61,13 +67,20 @@ var depsDevCmd = &cobra.Command{
 			viper.GetBool("use-csub"),
 			viper.GetBool("service-poll"),
 			viper.GetBool("retrieve-dependencies"),
-			args)
+			args,
+			viper.GetBool("enable-prometheus"),
+			viper.GetString("prometheus-address"),
```

Comment:
I think this should maybe be a port instead? since viper flags are global , there is less of a case of providing an address - and because this will be prone to misconfiguration.

Reply:
otherwise, nit use addr for consistency

Reply:
Sorry, I don't get it. Do you want to change the name?

Reply:
Just using a port here similar to `set.Int("csub-listen-port", 2782, "port to listen to on collect-sub service")` is what Brandon is saying here

Reply:
Yea, I don't think we have a use case of running this on a listening host that isn't 0.0.0.0 now and don't want to introduce that complexity rn

Reply:
👍